From: =?utf-8?q?Michal_Koutn=C3=BD?= <mkoutny@suse.com>
Date: Wed, 24 Jun 2020 12:43:22 +0200
Subject: missing: Add new Linux capabilities

Linux kernel v5.8 adds two new capabilities. Make sure we can recognize
them even when built with an older kernel.

(cherry picked from commit e41de5e491942b5391b1efb71c82ffd329b3d23d)
---
 src/basic/missing_capability.h | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/src/basic/missing_capability.h b/src/basic/missing_capability.h
index 1308a3d..dd6bccd 100644
--- a/src/basic/missing_capability.h
+++ b/src/basic/missing_capability.h
@@ -10,3 +10,19 @@
 #undef  CAP_LAST_CAP
 #define CAP_LAST_CAP   CAP_AUDIT_READ
 #endif
+
+/* 980737282232b752bb14dab96d77665c15889c36 (5.8) */
+#ifndef CAP_PERFMON
+#define CAP_PERFMON 38
+
+#undef  CAP_LAST_CAP
+#define CAP_LAST_CAP   CAP_PERFMON
+#endif
+
+/* a17b53c4a4b55ec322c132b6670743612229ee9c (5.8) */
+#ifndef CAP_BPF
+#define CAP_BPF 39
+
+#undef  CAP_LAST_CAP
+#define CAP_LAST_CAP   CAP_BPF
+#endif
